core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowResource.scala

```diff
@@ -86,6 +87,87 @@ object WorkflowResource {
 @Produces(Array(MediaType.APPLICATION_JSON))
 class WorkflowResource {
 
+  @GET
+  @Path("/ids")
+  def retrieveIDs(@Auth sessionUser: SessionUser): List[String] = {
+    val user = sessionUser.getUser
+    val workflowEntries = context
+      .select(WORKFLOW_USER_ACCESS.WID)
+      .from(WORKFLOW_USER_ACCESS)
+      .where(WORKFLOW_USER_ACCESS.UID.eq(user.getUid))
+      .fetch()
+
+    workflowEntries
+      .map(workflowRecord => workflowRecord.into(WORKFLOW_OF_USER).getWid().intValue().toString())
+      .toList
+  }
+
+  @GET
+  @Path("/owners")
+  def retrieveOwners(@Auth sessionUser: SessionUser): List[String] = {
+    val user = sessionUser.getUser
+    val workflowEntries = context
+      .select(USER.NAME)
+      .from(WORKFLOW_USER_ACCESS)
+      .join(WORKFLOW_OF_USER)
+      .on(WORKFLOW_USER_ACCESS.WID.eq(WORKFLOW_OF_USER.WID))
+      .join(USER)
+      .on(WORKFLOW_OF_USER.UID.eq(USER.UID))
+      .where(WORKFLOW_USER_ACCESS.UID.eq(user.getUid))
+      .groupBy(USER.UID)
+      .fetch()
+
+    workflowEntries
+      .map(workflowRecord => workflowRecord.into(USER).getName())
+      .toList
+  }
+
+  @GET
+  @Path("/searchOperators")
```

Comment:
backend api usually follow the format like `search-operators`, I suggest we change this to `search-by-operators`